packages/common/src/ui/layout/tables/columns/EditableQuantityColumn.tsx

```diff
@@ -48,7 +48,11 @@ export const getEditableQuantityColumn = <
 
     return (
       <BasicTextInput
-        sx={{ maxHeight: 40 }}
+        sx={{
+          maxHeight: 40,
+          paddingTop: '3px',
+          '& .MuiInput-input': { textAlign: 'right', width: '75px' },
+        }}
```

Comment:
Think a good target for `NumericTextInput` soon - adding the `type="number"` n stuff

Reply:
yep!